app/src/main/res/layout/activity_custom_selector.xml

```diff
@@ -14,6 +14,52 @@
       android:layout_width="match_parent"
       android:layout_height="0dp"
       app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintBottom_toTopOf="@id/bottom_layout"
       app:layout_constraintTop_toBottomOf="@+id/toolbar_layout"/>
 
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
I would prefer the bottom layout to be in a separate file and include it here same as it is done with the toolbar. 
Makes the XML more modular and easy to read. 

@nicolas-raoul views?

Reply:
Sounds like a good idea!